app/models/report_generators/data_quality/fy2017/q5.rb

```diff
@@ -8,15 +8,15 @@ module ReportGenerators::DataQuality::Fy2017
   class Q5 < Base
     ADULT = 18
     # PH = [3,9,10,13]
-    PH = GrdaWarehouse::Hud::Project::RESIDENTIAL_PROJECT_TYPES.values_at(:ph).flatten(1)
+    PH = HudUtility2024.residential_project_type_numbers_by_code.values_at(:ph).flatten(1)
     # TH = [2]
-    TH = GrdaWarehouse::Hud::Project::RESIDENTIAL_PROJECT_TYPES.values_at(:th).flatten(1)
+    TH = HudUtility2024.residential_project_type_numbers_by_code.values_at(:th).flatten(1)
     # ES = [1]
-    ES = GrdaWarehouse::Hud::Project::RESIDENTIAL_PROJECT_TYPES.values_at(:es).flatten(1)
+    ES = HudUtility2024.residential_project_type_numbers_by_code.values_at(:es).flatten(1)
```

Comment:
Should the reports tied to specific date ranges refer to a different HudUtility? I think it is fine, but ES is now [0, 1]. There shouldn't be any '0' in the DB, but it seems odd that a FY2017 report refers to the 2024 standards...